src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -382,6 +424,26 @@ class RideHailAgent(
 
   override def logPrefix(): String = s"RideHailAgent $id: "
 
+  def requestChargedParking(): Unit = {
+    val rideHailAgentLocation =
+      RideHailAgentLocation(vehicle.driver.get, vehicle.id, vehicle.beamVehicleType.id, vehicle.spaceTime, geofence)
+    val destinationUtm = rideHailAgentLocation.currentLocationUTM.loc
+    val beta1 = 1
+    val beta2 = 1
+    val beta3 = 0.001
+    val mnlUtilityFunction: Map[String, Map[String, UtilityFunctionOperation]] = Map(
+      "ParkingSpot" -> Map(
+        "energyPriceFactor" -> UtilityFunctionOperation("multiplier", -beta1),
+        "distanceFactor"    -> UtilityFunctionOperation("multiplier", -beta2),
+        "installedCapacity" -> UtilityFunctionOperation("multiplier", -beta3)
+      )
+    )
+    val mnl = MultinomialLogit[String, String](null)
+    val chargingInquiry = ChargingInquiry(Some(mnl), plugData = None, vehicle, vot = 0.0)
```

Comment:
@robfitzgerald @johanneshiry This is mostly a copy/paste from ChoosesParking. I know we had discussed the vot, but I wasn't sure how to work it so that fastcharging was preferred per @colinsheppard

Reply:
@JustinPihony we are actually modifying MNL, ParkingInquiry, and ChargingInquiry today as part of our integration of these things. it's different since an hour ago, so, maybe pull down the changes for now.

for prioritizing fast charging, perhaps we could add a term which is the "duration until full" cost, and, for RHA, set that multiplier higher, so it dominates over the other choices. i wonder what @johanneshiry thinks of that idea.

i'll be busy for some bits of the middle of today but i'll try and keep up with the discussion.